src/main/scala/cromwell/engine/backend/sge/SgeBackendCall.scala

```diff
@@ -19,7 +19,7 @@ case class SgeBackendCall(backend: SgeBackend,
   val stderr = callRootPath.resolve("stderr")
   val script = callRootPath.resolve("script.sh")
   val returnCode = callRootPath.resolve("rc")
-  val engineFunctions: SgeEngineFunctions = new SgeEngineFunctions(callRootPath, stdout, stderr, workflowDescriptor.ioInterface)
+  val engineFunctions: SgeEngineFunctions = new SgeEngineFunctions(callRootPath, stdout, stderr, workflowDescriptor.ioManager)
```

Comment:
`IoManager` is a bit strange in a sense that even though it `extends IoInterface` it doesn't really "implement" it, it uses actual `IoInterface` implementations (GCS, Shared FileSystem) and redirect to the correct one depending on the shape of the path.
The name `ioManager` was an attempt to capture this oddity, but I'm fine calling this an ioInterface if it's less confusing.